WebGL.md

```diff
@@ -0,0 +1,74 @@
+# Implementing Immutable Unity SDK for WebGL
+
+This guide provides focused instructions for implementing the Unity-Immutable-SDK in WebGL projects.
+
+For general information on the SDK, please refer to
+the [Immutable Unity SDK documentation](https://docs.immutable.com/sdks/zkEVM/unity).
+
+Live example can be found at https://immutable.github.io/unity-immutable-sdk/sample/webgl
+
+## WebGL Template Setup
+
+WebGL template is a configuration setting that lets you control the appearance of the HTML page, so that you can: test, demonstrate, and preview your WebGL application in a browser.
+
+A WebGL Template is always needed for creating a WebGL application and it will always be stored within **Assets > WebGL Templates** to be used. You can refer to [Unity: WebGL Templates](https://docs.unity3d.com/Manual/webgl-templates.html) for more information.
+
+A Custom WebGL Template is required to implement the Immutable Unity SDK in WebGL projects. The easiest way to create a new custom WebGL template is to make a copy of the built-in Default or Minimal templates, which are stored in corresponding subfolders under <Unity Installation> > PlaybackEngines > WebGLSupport > BuildTools > WebGLTemplates.
+
+Every Unity Project includes these templates by default. Copy a template and place it in your own **Assets > WebGLTemplates** folder, and rename it to something meaningful so you can identify your template later.
+
+Once you have created your own template, copy the following files from Passport package into the **Assets > WebGLTemplates** folder:
+   - `Packages/Immutable Passport/Runtime/Assets/WebGLTemplates/unity-webview.js`
+   - `Packages/Immutable Passport/Runtime/Assets/WebGLTemplates/callback.html`
+   - `Packages/Immutable Passport/Runtime/Assets/WebGLTemplates/logout.html`
+   - `Packages/Immutable Passport/Runtime/Assets/WebGLTemplates/index.html` > `Passport/index.html`
+
+3. Add the following script tag to the `index.html` in WebGL Templates:
+   ```html
+   <script src="unity-webview.js"></script>
+   ```
+
+## PKCE Login and Logout Implementation
+> [!IMPORTANT]  
+> WebGL supports only PKCE flow. Follow these steps for implementation:
```

Comment:
"Follow these steps for implementation:" shouldn't be part of the Important block right?